.velocitas.json

```diff
@@ -0,0 +1,15 @@
+{
+    "packages": [
+        {
+            "name": "devenv-runtimes",
+            "version": "v2.2.6"
```

Comment:
Why do we need the runtimes here?

Reply:
We already had it before (removed the install dapr script, mqtt broker config in settings,...), so I replaced the old style with the cli

Reply:
Yes, your're right. Wasn't aware of that any more ...